README.md

```diff
@@ -1,145 +1,43 @@
-# Composer template for Drupal projects
+## Build Status
 
 [![Build Status](https://travis-ci.org/drupal-composer/drupal-project.svg?branch=8.x)](https://travis-ci.org/drupal-composer/drupal-project)
```

Comment:
Could update this to resolve this too: https://www.drupal.org/project/guardr/issues/3071904

`[![Build Status](https://travis-ci.com/guardrdistro/guardr-project.svg?branch=8.x)](https://travis-ci.com/guardrdistro/guardr-project)`

Reply:
I don't see the build status code has been udpated

Reply:
I see them both there.  There isn't any explanation for the two different ones though.  Maybe I'm misunderstanding what you're looking for here.  http://astrocling.mediacurrent.s3.amazonaws.com/Screen-Shot-2019-08-02-17-07-50.74.png is what I'm seeing in my fork.

Reply:
yeah there should just be one, the one for guardr-project. Reasoning is to show the build status of this project. The drupal-project one points to the drupal-project build status at https://github.com/drupal-composer/drupal-project. I think I am going to merge this and then fix up that one piece.

Reply:
Gotya. I misunderstood what you were wanting. I can fix it and push the change. That’s easy.